lambda/src/app/confirmation-modal/confirmation-modal.component.html

```diff
@@ -1,23 +1,13 @@
-<ng-template [ngIf]="isActive">
-  <div class="sf-modal sf-modal--min" [ngClass]="{'is-active': isActive===true}" role="dialog" aria-labelledby="confirm" (click)="$event.stopPropagation()">
-    <div class="tn-modal" role="document">
-      <div class="tn-modal__content">
-        <div class="tn-modal__header">
-          <button class="tn-button tn-button--text tn-button--icon tn-button--small" aria-label="close" (click)="close($event)">
-            <span class="tn-icon tn-icon--close" role="presentation"></span>
-          </button>
-          <h4 class="tn-modal__title">{{ title }}</h4>
-        </div>
-        <div class="tn-modal__body">
-          <div class="tn-form__group sf-modal--messages">
-            {{ message }}
-          </div>
-        </div>
-        <footer class="tn-modal__footer-items">
-          <button class="tn-modal__button-primary sf-button--primary tn-button--small" (click)="ok($event)">OK</button>
-          <button class="tn-modal__button-secondary tn-button--small" (click)="close($event)">Cancel</button>
-        </footer>
-      </div>
-    </div>
-  </div>
-</ng-template>
+<fd-modal #confirmationModal role="dialog" aria-labelledby="confirm" (click)="$event.stopPropagation()">
```

Comment:
This will remove the border on 'x' in the modal.
```suggestion
<fd-modal #confirmationModal role="dialog" class="y-fd-modal" aria-labelledby="confirm" (click)="$event.stopPropagation()">
```